node2vec/node2vec.py

```diff
@@ -174,8 +174,8 @@ def fit(self, **skip_gram_params) -> gensim.models.Word2Vec:
         if 'workers' not in skip_gram_params:
             skip_gram_params['workers'] = self.workers
 
-        if 'size' not in skip_gram_params:
-            skip_gram_params['size'] = self.dimensions
+        if 'vector_size' not in skip_gram_params:
+            skip_gram_params['vector_size'] = self.dimensions
```

Comment:
I'd rather not change the argument in the `fit` method. but rather check for which version of `gensim` is installed, and passing it to `size` or `vector_size` depending on the version

Reply:
Sure, that would work. I changed it in my local file anyway. I'm still looking forward to the update version. Thank you for looking into it.